MIGRATION.md

```diff
@@ -0,0 +1,267 @@
+# Migrate from version < v13.0.0
+
+This guide illustrates differences between Lob’s the legacy versions of this SDK and the new current version.
+
+In this guide we compare how `v12.*.*` and >=`v13.0.0` implement the following method pattern.
+
+- CREATE
+- LIST
+- GET
+- DELETE
+- VERIFY (BANK ACCOUNTS)
+- UPDATE (TEMPLATES)
+
+## INSTALL
+
+Similar to the legacy version, simply update your POM file dependency as follows:
+
+```xml
+<dependency>
+    <groupId>com.lob</groupId>
+    <artifactId>lob-java</artifactId>
+    <version>13.0.0</version>
```

Comment:
We talked about it, not sure where we landed on having this be <CURRENT_VERSION_OF_JAVA> or something along these lines.